Taskfile.yml

```diff
@@ -318,6 +318,25 @@ tasks:
     cmds:
       - cargo insta test --accept --lib -- --quiet
 
+  test-rust-coverage:
+    desc: Run tests with instrumentation for code coverage
+    env:
+      # https://github.com/mozilla/grcov#generate-a-coverage-report-from-coverage-artifacts
+      CARGO_INCREMENTAL: 0
+      RUSTFLAGS: "-Cinstrument-coverage"
+      LLVM_PROFILE_FILE: "cargo-test-%p-%m.profraw"
+      # Use a different target dir so we don't poison the cache
+      CARGO_TARGET_DIR: "./target-cov"
+    cmds:
+      # Setup
+      - cargo install grcov
+      - rustup component add llvm-tools-preview
```

Comment:
I'd ask for these setup tasks not be present here, as this does not work on my machine.